src/containers/Polls/index.tsx

```diff
@@ -86,7 +87,8 @@ function PollsInfo(props) {
         Promise.all(
           polls.map((poll: any) => {
             if (poll.plurality && poll.participation) {
-              return Promise.resolve(poll)
+              const participation = new BigNumber(poll.participation)
+              return Promise.resolve({ ...poll, participation: participation.dp(4).toString() })
```

Comment:
Wouldn't this only work for chached polls? So if the poll isn't cached we need to do something similar in line 94

Reply:
I also added the `dp(4)` in the `getParticipation` function if that's what you meant.

Reply:
You are right, github didn't show me the function name...
Then I think we don't need this change. It may have been not working because you already had cached data with more than 4 decimals

Reply:
that was it, clearing the cache did the trick.